### ISSUE TITLE
Android JCenter - build failure

### ISSUE THREAD

**Issue by dawidchyrzynski** [maxkomarychev/react-native-ultimate-config#111]
This library still relies on the jcenter repository. It should be migrated to maven central.

```
FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':react-native-ultimate-config:verifyReleaseResources'.
> Could not resolve all task dependencies for configuration ':react-native-ultimate-config:releaseRuntimeClasspath'.
   > Could not resolve com.facebook.react:react-native:+.
     Required by:
         project :react-native-ultimate-config
      > Failed to list versions for com.facebook.react:react-native.
         > Unable to load Maven meta-data from https://jcenter.bintray.com/com/facebook/react/react-native/maven-metadata.xml.
            > Could not HEAD 'https://jcenter.bintray.com/com/facebook/react/react-native/maven-metadata.xml'.
               > Read timed out
```

**Comment by vagnerlandio** [comment]
Solution here -> https://github.com/maxkomarychev/react-native-ultimate-config/issues/107#issuecomment-1307928241

**Comment by maxkomarychev** [comment]
There was a massive refactoring made to the library and this problem could have been fixed in a new version 4.1.0-alpha.0 (https://github.com/maxkomarychev/react-native-ultimate-config/pull/128)

Please give it a try!! Thank you!

**Comment by maxkomarychev** [comment]
I'm going to close this ticket for now. Please feel free to reopen it if issue persists in the most recent version. Thank you!
